CameraXBasic/app/build.gradle

```diff
@@ -40,6 +39,10 @@ android {
         jvmTarget = rootProject.ext.java_version
     }
 
+    buildFeatures {
```

Comment:
is there a reason we want to enable viewBinding even though this PR doesn't include using the view binding features?

Reply:
yeah, that is true: it was left from local experiment. will removed in the next patch.  Thanks

Reply:
removed now (will change all to view binding next time), please have one more look. Thanks!